null_safety_examples/strong/lib/bounded/my_collection.dart

```diff
@@ -1,4 +1,4 @@
-// ignore_for_file: sort_constructors_first
+// ignore_for_file: strict_raw_type
```

Comment:
It is not documented anywhere on dart.dev, but updating the analysis option documentation, including adding these (newer) strict modes is on my list of work to get done soon. You can find some documentation about `strict-raw-types` on the language repo(https://github.com/dart-lang/language/blob/master/resources/type-system/strict-raw-types.md). I've enabled it as specifying explicit type arguments is often a good practice to avoid dynamic and prevent potential type confusions.

An issue tracking the documentation of this on the website: https://github.com/dart-lang/site-www/issues/2338